src/deploy-certificate.sh

```diff
@@ -58,10 +58,10 @@ then
     exit 1 
 fi
 
-arm_template_file="nginx-for-azure-certificate-template.json"
+arm_template_file="nginxaas-for-azure-certificate-template.json"
 
 #get the ARM template file
-wget -O "$arm_template_file" https://raw.githubusercontent.com/nginxinc/nginx-for-azure-deploy-action/a69d33feaa1a8a012ec44c138ca78c6ec4db9f29/src/nginx-for-azure-certificate-template.json
+wget -O "$arm_template_file" https://raw.githubusercontent.com/nginxinc/nginxaas-for-azure-deploy-action/a69d33feaa1a8a012ec44c138ca78c6ec4db9f29/src/nginxaas-for-azure-certificate-template.json
```

Comment:
We should avoid renaming the repo or any references to it for now. We can take that on separately if the current repo name becomes a source of confusion for users.